weejson/jackson/src/com/rallyhealth/weejson/v0/jackson/WeeJackson.scala

```diff
@@ -0,0 +1,29 @@
+package com.rallyhealth.weejson.v0.jackson
+
+import java.io.InputStream
+
+import com.fasterxml.jackson.core.JsonParser
+import com.rallyhealth.weejson.v0.Readable
+import com.rallyhealth.weejson.v0.jackson.DefaultJsonFactory.Instance
+import com.rallyhealth.weepickle.v0.core.Visitor
+
+object WeeJackson {
+
+  def parse(s: String): Readable = parse(Instance.createParser(s))
+
+  def parse(in: InputStream): Readable = parse(Instance.createParser(in))
+
+  def parse(in: Array[Byte]): Readable = parse(Instance.createParser(in))
+
+  def parse(parser: JsonParser): Readable = new Readable {
+
+    override def transform[T](f: Visitor[_, T]): T = {
+      var result: T = null.asInstanceOf[T]
+      val generator = VisitorJsonGenerator.withCallback(f)(result = _)
+      generator.consume(parser)
+      parser.close()
```

Comment:
```suggestion
      Try(parser.close())
```
I don't think you want an Exception in these `close()` calls to break parsing.

Reply:
Gah, we have this conversation every time! Thank you for being my `catch` block.